### ISSUE TITLE
Val Loss increases

### ISSUE THREAD

**Issue by csccsccsccsc** [histocartography/hact-net#2]
When I train HACT-Net (or the cell graph / tissue graph net), the validation loss keeps increasing and the accuracy keeps unchanged (very low).

**Comment by guillaumejaume** [comment]
I would need more information to help you here, 

- Are you re-training on the BRACS dataset?, If yes, BRACSv1 or BRACSv2 (cf. README, Section: Downloading the BRACS dataset)
- Have you tried to use a pre-trained model to see if you can reproduce the results?
- Have you tried sufficient hyper-parameter search?, ie learning rate, batch size, etc

**Comment by csccsccsccsc** [comment]
1) I train on the BRACSv1 dataset (".../BRACS_RoI/previous_versions/Version1_MedIA").

2) I try the pre-trained model, but I got:
Test weighted F1 score 0.4220916765247371
Test accuracy 0.46166134185303515

3) I have tried learning rate 0.1 / 0.001 / 0.0005 / 0.0001 / 0.00001. I only wait for around 10 epochs. In my experiments, the "val weighted F1 score" and the "val accuracy" keeps unchanged (usually around 0.04527802507158154 and 0.16220735785953178).

**Comment by csccsccsccsc** [comment]
Will this caused by the different graph features? Could you send me your processed graph features?

**Comment by csccsccsccsc** [comment]
Sorry. It seems that my problem is caused by the error during downloading datasets.

**Comment by csccsccsccsc** [comment]
I

**Comment by csccsccsccsc** [comment]
I found that there exist some empty images in the website:
train/5_DCIS/BRACS_1478_DCIS_24.png, train/5_DCIS/BRACS_1478_DCIS_30.png, train/5_DCIS/BRACS_295_DCIS_31.png
train/6_IC/BRACS_280_IC_14.png
val/6_IC/BRACS_1638_IC_13.png

**Comment by guillaumejaume** [comment]
Thanks for reporting the issue with empty images. We're working on it, and will fix it as soon as possible.

**Comment by guillaumejaume** [comment]
You can download the preprocessed cell, tissue and hact graphs for the BRACS dataset here:
- https://ibm.box.com/s/6v4sasavltjzi91gmohswz2ek6i8i3lp

Or by downloading this zip file that includes the test cell graphs:
-  https://ibm.box.com/shared/static/412lfz992djt8u6bgu13y9cj9qsurwui.zip

**Comment by guillaumejaume** [comment]
Empty images updated on the FTP server, thanks for reporting the issue.
